docs/arcadia-users-group/20230228-intro-to-python-1/problem_set_1_answers.ipynb

```diff
@@ -0,0 +1,607 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    def factor(a, b, c):
</pre><p>Is this the solution? Doesn't seem different</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arcadia-Science/arcadia-computational-training/pull/50/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Whoops! I'm going to remove this question, but I definitely forgot to save the solution.